testing/distributions-integ-tests/src/integTest/groovy/org/gradle/DistributionIntegrationSpec.groovy

```diff
@@ -155,7 +157,7 @@ abstract class DistributionIntegrationSpec extends AbstractIntegrationSpec {
         expect:
         def size = getZip().size()
 
-        assert size <= getMaxDistributionSizeBytes(): "Distribution content needs to be verified. If the increase is expected, raise the size by ${Math.ceil((size - getMaxDistributionSizeBytes()) / 1024 / 1024)}"
+        assert size <= getMaxDistributionSizeBytes() : "Distribution content needs to be verified. If the increase is expected, raise the size by ${Math.ceil((size - getMaxDistributionSizeBytes()) / 1024 / 1024)}"
```

Comment:
💅  Wasn't this formatting changed in the previous PR as well?
I wonder if it's unstable for some reason, or just an artifact of a merge/rebase?

Reply:
right, that is weird, auto formatting removes it actually.